ui/ducks/metamask/metamask.js

```diff
@@ -272,8 +273,10 @@ export function getNftsDropdownState(state) {
 
 export const getNfts = (state) => {
   const {
-    metamask: { allNfts, selectedAddress },
```

Comment:
If `state.metamask.selectedAddress` doesn't represent the current address, do we know why?

Reply:
Certainly a bug, while we have this transition from the `PreferencesController` to the `AccountsController`, the data should remain consistent

Reply:
Yes, its a bug they should still be in sync while the migration is happening.